packages/ui-core/src/components/slider/slider.utils.tsx

```diff
@@ -0,0 +1,65 @@
+import { Position } from '../../types';
+
+export const calculatePercentage = (
+  x: number,
+  sliderSize: number,
+  max: number,
+  min: number,
+  step?: number
+) =>
+  Math.round(
+    Math.round((((x * 100) / sliderSize / 100) * (max - min)) / step) * step
+  );
+
+export const colorsString = (colors: string[], colorSteps: number) => {
+  const newColors = colors.slice(0, colorSteps);
+  return newColors.join(', ');
+};
+
+export const onChangeValue = (
+  controls: number,
+  valMin: number,
+  valMax: number
+) => (controls === 2 ? { min: valMin, max: valMax } : valMin);
+
+export const calculateValueStep = (x: number, step: number) =>
+  Math.round(x / step) * step;
+
+export const calculateTicks = (
+  min: number,
+  max: number,
+  steps: number,
+  sliderSize: number
+) => {
+  const ticks = [];
+  const step = steps
+    ? sliderSize / steps > 35
```

Comment:
What is `35` number ? Could You please move it to `const` variable with self-descriptive name please ?